src/Plugin/migrate/process/ValidEDTF.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Drupal\migrate_islandora_csv\Plugin\migrate\process;
+
+use Drupal\Component\Plugin\ConfigurableInterface;
+use Drupal\controlled_access_terms\EDTFUtils;
+use Drupal\migrate\MigrateExecutableInterface;
+use Drupal\migrate\MigrateSkipRowException;
+use Drupal\migrate\ProcessPluginBase;
+use Drupal\migrate\Row;
+
+/**
+ * Skips processing the current row when the EDTF date isn't valid.
+ *
+ * Available configuration keys:
+ * - intervals (optional): Boolean of whether this field is supporting intervals
+ *   or not, defaults to TRUE.
```

Comment:
My testing shows the default is actually FALSE. When I leave it unset
```yml
  field_cat_date_end:
    plugin: valid_edtf
    source: date
```
 I get the message:
```
 ----------------------- ------------------- ------- -------------------------------------------------------------------------------------------------------------- 
  Source ID(s)            Destination ID(s)   Level   Message                                                                                                       
 ----------------------- ------------------- ------- -------------------------------------------------------------------------------------------------------------- 
  http://fake.uri/fake1                       4       The value: 2020-04-17/2020-04-29 is not a valid EDTF date: Could not parse the date '2020-04-17/2020-04-29'.  
 ----------------------- ------------------- ------- -------------------------------------------------------------------------------------------------------------- 
```
Whereas 
```yml
  field_cat_date_end:
    plugin: valid_edtf
    source: date
    intervals: TRUE
```
successfully saves the record.

Reply:
Will have a look, thanks @seth-shaw-unlv

Reply:
As noted in https://www.drupal.org/project/drupal/issues/2937177#comment-12489678 this is something that _should_ be done by default but is blocked by that issue I linked and https://www.drupal.org/project/drupal/issues/2852463.

As a stop gap taking the approach of still using `ConfigurableInterface` but making sure the `defaultConfiguration` still gets applied. For a similar approach albeit not Migrate: https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Action%21ConfigurableActionBase.php/class/ConfigurableActionBase/8.9.x.

Reply:
The commit resolved this, but sets is suffering the same issue intervals did.